### DISCUSSION TITLE
Sort the sources list by different criteria

### DISCUSSION THREAD

**Discussion by wyattford** [wstyres/Zebra#1060] [Ideas]
**Is your feature request related to a problem? Please describe.**
It gets frustrating when I add a new source and am unable to find the source in the listing because the title isn't always the same as the actual URL of the repo.

**Describe the solution you'd like**
I'd like to be able to sort my sources by the url instead of the name of the source.

**Describe alternatives you've considered**
A alternative solution would be to have different options to sort the source list by such as by date of adding, the amount of tweaks, etc.

**Comment by p0358** [comment]
+1, sorting via name, URL, amount of **installed** packages
and also some counter that would display the number of installed packages on the list itself

**Comment by demhademha** [comment]
+1 on that idea

**Comment by worldlydev** [comment]
What about the sources page using the same filters as the packages page? Date (added), name (alphabetical), and size (amount of packages).

A tweak counter could also be added like I requested in #956. Would be so awesome
